index.html

```diff
@@ -1,22 +1,225 @@
-      crossorigin="anonymous"
-    />
+    <title> Portfolio Website</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
   </head>
-  <body>
-    <div class="container"></div>
-    <script
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"
-      integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"
-      crossorigin="anonymous"
-    ></script>
+ <body>
+    <div id="home-page"> 
+      <header class="header">
+        <!--  -->
+        <nav class="navbar ">
+          <div class="container-fluid">
+            <a class=" logo"  href="#">
+              
+              My Logo
+            </a>
+          </div>
+        </nav>
+         <!-- -->
+          <!-- <a href="#" class="logo">My Logo</a> 
+          <nav id="nav-hamburger">
+          <img src="images/nav.png">        
+          </nav> -->
+      </header>
+      <section id="main">
+        <div>
+          <h2><span class="main-heading">I'm Celyne.<br>Glad to See you!</span></h2>
+          <p class="main-paragraph">I'm a software developer! I can help you build a product, feature or website Look through
+               some of my work and experience! If you like what you see and have 
+               a project you need coded, don't hesitate to contact me.</p>
+               <label class="connect">LET'S CONNECT</label>
+              </div>
+       <div id="socials">
+          <ul class="social-media-list">
+            <li><a href="#" class="icons"><img src="images/twitter.png"></a></li>
+            <li><a href="#" class="icons"><img src="images/Linkedin.png"></a></li>
+            <li><a href="#" class="icons"><img src="images/M.png"></a></li>
+            <li><a href="#" class="icons"><img src="images/github.png"></a></li>
+            <li><a href="#" class="icons"><img src="images/hi 5.png"></a></li>
+          </ul><br>
+          </div>
+        </section>
+    </div>
+    <div id="works-grid">
+     <div id="works-1">
+        <div class="img-container"><img src="images/Snapshoot.png"></div>
+        <div class="img-container-desktop"><img class="desktop-image" src="images/Snapshoot-Desktop1.png" alt=""></div>
+        <div class="works-contents">
+          <h2 class="works-heading">Tonic</h2>
+          <div class="works-frame">
+            <label class="frame-text1">CANOPY</label>
+            <div class="frame-counter"><img src="images/Counter.png"></div>
+            <label class="frame-text2">Back End Dev</label>
```

Comment:
As stated by the requirements, you could use hear a bootstrap classe for the card, this will help for responsivness